app/controllers/admin/custom_fields_controller.rb

```diff
@@ -160,7 +159,9 @@ def update
     params[:custom_field][:min] = ParamsService.parse_float(params[:custom_field][:min]) if params[:custom_field][:min].present?
     params[:custom_field][:max] = ParamsService.parse_float(params[:custom_field][:max]) if params[:custom_field][:max].present?
 
-    custom_field_entity = build_custom_field_entity(@custom_field.type, params[:custom_field])
+    custom_field_entity = build_custom_field_entity(@custom_field.type, params[:custom_field]).merge(
+      sort_priority: @custom_field.sort_priority
+    )
```

Comment:
It feels weird that sort_priority is merged in here when all other fields are set inside constructor build_custom_field_entity. Makes it harder to follow the code / debug.